### DISCUSSION TITLE
Use with system version of direnv (MacOS arm)

### DISCUSSION THREAD

**Discussion by comfortablynick** [asdf-community/asdf-direnv#68] [Q&A] [answered]
Is it possible to get this working with the system (homebrew) version of direnv? I've built my whole config around this and I can't seem to get it working on my M1 Macs.

This is what I get when loading ~/.envrc (containing only `use asdf`):

```
direnv: loading ~/.envrc
direnv: using asdf
direnv: referenced  does not exist
direnv: export -LD_LIBRARY_PATH
```

Looking at the source code for asdf-direnv I think this is the issue:

https://github.com/asdf-community/asdf-direnv/blob/e96ecf374cb9588c7b400f991254814e3226eefb/lib/commands/command.bash#L25

```
$ asdf where direnv
System version is selected
```

Is it not possible for asdf-direnv to create a folder for the .envrc file caches in this case? I don't think there's any way for me to install direnv right now using asdf.

**Comment by smorimoto** [comment, marked as answer]
The easiest way is to install it using Homebrew, etc, and specify “system” as the version. See #58.
